create_test_cluster.py

```diff
@@ -79,6 +98,7 @@ def check_args() -> Namespace:
                            "Otherwise the provided name will be used",
                       )
   parser.add_argument('--debug', '-d', action='store_true', required=False, help="Will print additional debug statements (e.g. output from all run commands)")
+  parser.add_argument('--prometheus', '-prom', action='store_true', required=False, help="Will install the Prometheus operator for scraping metrics.")
```

Comment:
I know `-p` is taken so I suggest to use some unrelated single letter argument (`-m` for monitoring) because all others either specify the full name or just one letter.
```suggestion
  parser.add_argument('--prometheus', '-m', action='store_true', required=False, help="Will install the Prometheus operator for scraping metrics.")
```